csaf_2.0/prose/csaf-v2-editor-draft.md

```diff
@@ -2932,6 +2934,8 @@ Example which fails the test:
 
 > The required element `baseSeverity` is missing.
 
+> A tool MAY add one or more of the missing properties `version`, `baseScore` and `baseSeverity` based on the values given in `vectorString` as quick fix.
```

Comment:
The prior wording `implement add one or more of …` read like copy-pasta (which I am sure it was not, but …) 
A sentence with a `MAY` Is often hard to read especially when additional layers of `potential only application`  join that keyword.
My understanding is, that in this and similar phrases about tools (proxy actors) a capability of the proxy is given permission to exist per this spec but that these phrases IMO try to achieve too much.
I propose to rely on the fearsome power of the `MAY` and state the possible actions without any `implements` addition (just removing the word).

Reply:
> The prior wording `implement add one or more of …` read like copy-pasta 

You are completely right.

> (which I am sure it was not, but …)

Sorry about that - but it looks like it was :wink: 

> I propose to rely on the fearsome power of the `MAY` and state the possible actions without any `implements` addition (just removing the word).

Thank you for the suggestion. It makes it much easier to read. There was only one (`implement the recommendation`) which I was not sure how to replace that one... Do you have an idea or is that fine as it doesn't have multiple layers?

Reply:
W.r.t. an enhancement for `implement the recommendation` I suggest to de-noun-ify and try to form a statement starting with `MAY recommend …` taking the perspective of the tool changing, or if the `recommendation` Is a jungle of interwoven statements, maybe flatten into a list of possible actions.

If the `recommendation` Is not actionable, this will not work, but then how shall the tool implement?

I hope this helps.

Reply:
Sure. Possibly solved in 531e172.